source/content/guides/autopilot/02-enable-autopilot.md

```diff
@@ -10,7 +10,7 @@ showtoc: true
 anchorid: enable-autopilot
 permalink: docs/guides/autopilot/enable-autopilot/
 editpath: autopilot/02-enable-autopilot.md
-reviewed: "2021-08-10"
+reviewed: "2022-04-05"
 ---
 
 ## Enable Autopilot
```

Comment:
We're missing some structure in this section. Can we add headers for the 3 steps in the onboarding process (similar to how the "Autopilot Configuration - Manage Autopilot Settings" is laid out)? Also Environment Syncing is an option during onboarding, just as it is in the Configuration Settings. So we need steps and screenshots for that. Let me know if you have trouble finding this in the Figma file i sent you.

Reply:
@kelly-mcgonnigle Can you clarify which 3 steps you're referring to?

Reply:
Yep! I explain in this Loom. let me know if you have follow up qs: https://www.loom.com/share/d8387d3e112540dca896b6e44689c8b8